doc/api/tls.md

```diff
@@ -658,6 +674,7 @@ certificate.
 ```
 
 If the peer does not provide a certificate, an empty object will be returned.
+If the socket has been destroyed `null` will be returned.
```

Comment:
I might be wrong, but I think there needs to be a comma after `destroyed`? (above as well?)

Reply:
```suggestion
If the socket has been destroyed, `null` will be returned.
```